<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PivotFilters</class>
 <widget class="QWidget" name="PivotFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>133</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Operator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Operator">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>And</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Or</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Operator2"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Field</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Field"/>
       </item>
       <item>
        <widget class="QComboBox" name="Field2"/>
       </item>
       <item>
        <widget class="QComboBox" name="Field3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Condition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Condition">
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>==</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>!=</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Condition2"/>
       </item>
       <item>
        <widget class="QComboBox" name="Condition3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Value"/>
       </item>
       <item>
        <widget class="QComboBox" name="Value2"/>
       </item>
       <item>
        <widget class="QComboBox" name="Value3"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
